posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -576,6 +576,17 @@
   
   '
 ---
+# name: test_create_table_query[ga4_channel_definition]
+  '
+  
+  CREATE TABLE IF NOT EXISTS ga4_channel_definition ON CLUSTER posthog (
```

Comment:
Should we call this `ga4`? Very likely we'll modify, adapt and posthogify this list.